src/features/preview/PreviewLayer.js

```diff
@@ -275,8 +275,9 @@ const PreviewLayer = (ownProps) => {
           ref={trRef}
           centeredScaling={true}
           resizeEnabled={!props.layer.trackEnabled}
+          rotateEnabled={props.layer.maskMachine !== 'circle' }
```

Comment:
Should we be pushing these into the layer/shapes? Some default and then the circle mask would be doing something unique, instead of putting circle mask specific code here?